tools/arch/cockpit.install

```diff
@@ -0,0 +1,7 @@
+post_install() {
+  printf "# List of users which are not allowed to login to Cockpit\nroot\n" > /etc/cockpit/disallowed-users
```

Comment:
So that bit will only run on fresh installs, not on upgrades?

Shuold get same chmod as for rpm.

Reply:
Yes, this only runs on clean installs, for upgrades there is `post_upgrade`.